R/getBaselineDivergence.R

```diff
@@ -153,20 +150,21 @@ setMethod("getBaselineDivergence", signature = c(x = "SummarizedExperiment"),
         reference <- args[["reference"]]
         time_res <- .get_time_difference(x, time.col, reference)
         # Create a DF to return
-        res <- .convert_divergence_to_df(x, res, time_res, ...)
+        res <- .convert_divergence_to_df(x, res, reference, time_res, ...)
         return(res)
     }
 )
 
 #' @rdname addBaselineDivergence
 #' @export
 setMethod("addBaselineDivergence", signature = c(x = "SummarizedExperiment"),
-    function(x, name = "divergence", name.time = "time_diff", ...){
+    function(x, name = c("divergence", 
+                         "time_diff", "ref_samples"), ...){
```

Comment:
This line has weird indentation. This should be either like this

```
function(x, name = c("divergence", "time_diff", "ref_samples"),
    ...){ 
```
or if that does not fit, then
```

function(x,
    name = c("divergence", "time_diff", "ref_samples"), ...){
```

Reply:
or preferably you can format this similarly to get* function above